app/script/view_model/content/InputBarViewModel.js

```diff
@@ -645,13 +660,13 @@ z.viewModel.content.InputBarViewModel = class InputBarViewModel {
   sendMessage(messageText, replyMessageEntity) {
     if (messageText.length) {
       const mentionEntities = this.currentMentions.slice();
-      const quoteEntity = replyMessageEntity && new z.message.QuoteEntity(replyMessageEntity.id, '');
+      const replyEntity = replyMessageEntity && new z.message.QuoteEntity(replyMessageEntity.id, '');
```

Comment:
Let's keep it as the name of the class of the entity (`quoteEntity` which is an instance of `QuoteEntity`)